gradio/media_data.py

```diff
@@ -8648,3 +8648,7 @@
     "interactive": None,
     "visible": True,
 }
+BASE64_MICROPHONE = {
```

Comment:
I just recorded myself using a gradio demo and saved it before the back-end processed it.

It says "my name is Freddy" lol